Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -446,6 +444,125 @@ def getTupleSize_(flags, axisCount):
 			size += axisCount * 4
 		return size
 
+	def getDeltaType(self):
+		""" Check if deltas are (x, y) as in gvar, or single values as in cvar.
+		Returns a string ("gvar" or "cvar"), or None if empty.
+		"""
+		firstDelta = next((c for c in self.coordinates if c is not None), None)
+		if firstDelta is None:
+			return  # empty or has no impact
+		if type(firstDelta) is tuple and len(firstDelta) == 2:
+			return "gvar"
+		elif type(firstDelta) in (int, float):
+			return "cvar"
```

Comment:
would it make sense to return 1 or 2 [or 0 if no impact?], the number of active values, rather than tags?

Reply:
I renamed getDeltaType to getCoordWidth (returns 0, 1 or 2)